modules/http2/h2_stream.c

```diff
@@ -897,7 +900,14 @@ apr_status_t h2_stream_end_headers(h2_stream *stream, int eos, size_t raw_bytes)
      *      of CONNECT requests (see [RFC7230], Section 5.3)).
      */
     if (!ap_cstr_casecmp(req->method, "CONNECT")) {
-        if (req->scheme || req->path) {
+        if (req->protocol && !strcmp("websocket", req->protocol)) {
+            if (!req->scheme || !req->path) {
+                ap_log_cerror(APLOG_MARK, APLOG_INFO, 0, stream->session->c1,
+                              H2_STRM_LOG(APLOGNO(), stream, "Request to websocket CONNECT "
+                              "without :scheme or :path, sending 400 answer"));
+            }
```

Comment:
Hm, what is a websocket CONNECT? And why does it require a scheme and path?
Also, it seems it's not really "sending 400 answer" here. (not set_error_response() like in the other error cases below).

Reply:
RFC 8441 defines how to bootstrap websockets over HTTP/2. Will check on the error response setting.